components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/APGARController.java

```diff
@@ -108,8 +108,10 @@ public void save(Patient patient)
                 Integer value = data.get(propertyName);
                 BaseProperty<ObjectPropertyReference> field =
                     (BaseProperty<ObjectPropertyReference>) dataHolder.getField(propertyName);
-                if (field != null && value != null) {
+                if (value != null) {
```

Comment:
@marta, is that right that `field` can not be null unless `dataHolder`, which is `doc.getXObject(Patient.CLASS_REFERENCE)` is incorrect and does not have the correct `propertyName` (which is supposedly should, if the version of the code matches the version of the classes in DB)? In which case we probably do want a `NullPointerException` so that we see there is a problem in the installation (class mismatches code)